app/controllers/api/v1/people_controller.rb

```diff
@@ -111,11 +111,11 @@ def creator
       end
 
       def updater
-        @updater ||= People::Updater.new(params[:id], person_params)
+        @updater ||= People::Updater.new(person, person_params)
       end
 
       def person
-        @person ||= Person.find(params[:person_id])
+        @person ||= Person.find(params[:person_id] || params[:id])
```

Comment:
why does the id change names between `:person_id` and `:id`? is the routing configured OK?

Reply:
Nested routes 😢

Reply:
```                     api_v1_person_images GET    /api/v1/people/:person_id/images(.:format)                                               api/v1/people#image
                api_v1_person_court_cases GET    /api/v1/people/:person_id/court_cases(.:format)                                          api/v1/people#court_cases
                  api_v1_person_timetable GET    /api/v1/people/:person_id/timetable(.:format)                                            api/v1/people#timetable
                            api_v1_people GET    /api/v1/people(.:format)                                                                 api/v1/people#index
                                          POST   /api/v1/people(.:format)                                                                 api/v1/people#create
                            api_v1_person PATCH  /api/v1/people/:id(.:format)                                                             api/v1/people#update
                                          PUT    /api/v1/people/:id(.:format)                                                             api/v1/people#update```

Reply:
Boo :-(